openmdao/utils/om.py

```diff
@@ -563,87 +608,98 @@ def _post_setup_exec(options):
                   "Display the call tree for the specified class method and all 'self' class "
                   "methods it calls."),
-    'iprof': (_iprof_setup_parser, _iprof_exec,
+    'check': (_check_config_setup_parser, _simple_exec, _check_config_cmd,
+              'Perform a number of configuration checks on the problem.'),
+    'cite': (_cite_setup_parser, _simple_exec, _cite_cmd,
+             'Print citations referenced by problem'),
+    'iprof': (_iprof_setup_parser, _iprof_exec, None,
               'Profiling of calls to particular object instances.'),
-    'iprof_totals': (_iprof_totals_setup_parser, _iprof_totals_exec,
+    'iprof_totals': (_iprof_totals_setup_parser, _iprof_totals_exec, None,
                      'Total timings of calls to particular object instances.'),
-    'mem': (_mem_prof_setup_parser, _mem_prof_exec, 'Memory profiler.'),
-    'mempost': (_mempost_setup_parser, _mempost_exec, 'Post-processor for memory profile output.'),
-    'view_coloring': (_view_coloring_setup_parser, _view_coloring_exec,
-                      'Colored jacobian viewer.'),
-    'xdsm': (_xdsm_setup_parser, _xdsm_cmd, 'XDSM viewer.'),
-    'scaffold': (_scaffold_setup_parser, _scaffold_exec,
+    'mem': (_mem_prof_setup_parser, _mem_prof_exec, None, 'Memory profiler.'),
+    'mempost': (_mempost_setup_parser, _mempost_exec, None,
+                'Post-processor for memory profile output.'),
+    'n2': (_n2_setup_parser, _n2_cmd, None, 'Display an interactive N2 diagram of the problem.'),
+    'partial_coloring': (_partial_coloring_setup_parser, _simple_exec, _partial_coloring_cmd,
+                         'Compute coloring(s) for specified partial jacobians.'),
+    'scaffold': (_scaffold_setup_parser, _scaffold_exec, None,
                  'Generate a simple scaffold for a component.'),
+    'summary': (_config_summary_setup_parser, _simple_exec, _config_summary_cmd,
+                'Print a short top-level summary of the problem.'),
+    'total_coloring': (_total_coloring_setup_parser, _simple_exec, _total_coloring_cmd,
+                       'Compute a coloring for the total jacobian.'),
+    'trace': (_itrace_setup_parser, _itrace_exec, None, 'Dump trace output.'),
+    'tree': (_tree_setup_parser, _simple_exec, _tree_cmd, 'Print the system tree.'),
+    'view_coloring': (_view_coloring_setup_parser, _view_coloring_exec, None,
+                      'Colored jacobian viewer.'),
+    'view_connections': (_view_connections_setup_parser, _simple_exec, _view_connections_cmd,
+                         'Connection viewer showing values and source/target units.'),
+    'view_mm': (_meta_model_parser, _simple_exec, _meta_model_cmd, "Meta Model Viewer."),
+    'view_model': (_n2_setup_parser, _view_model_cmd, None,
+                   'Display an interactive N2 diagram of the problem. '
+                   '(Deprecated, please use n2 instead.)'),
+    'xdsm': (_xdsm_setup_parser, _xdsm_cmd, None, 'XDSM viewer.'),
 }
```

Comment:
Asked Bret to make the grammatical structure more parallel in the command line help descriptions.

Reply:
fixed